JS-Algorithms/challenges.js

```diff
@@ -95,7 +137,15 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 ***** */
 
 const digitSum = (n) => {
-  // YOUR CODE HERE...
+  function factorial (x) {
```

Comment:
Nice! You're using recursion, what would happen for large numbers? Do you think we could have performance issues? If yes, what we could do? no need to write any code.

Reply:
This is an interesting question, to be honest I know a little bit about Big O notation, I guess this would be a 0(n) issue, so it would perfectly run into performance issues. I thinks another approach would be to make the function iterable. I'm not an expert about this topic yet.

But I will take into consideration in my knowledge path because I consider this is an important topic to know.